packages/libcunit/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://sourceforge.net/projects/cunit/
+TERMUX_PKG_DESCRIPTION="C Unit Testing Framework"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Thibault Meyer <meyer.thibault@gmail.com>"
+TERMUX_PKG_VERSION=2.1.3
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL=https://github.com/Linaro/libcunit/releases/download/2.1-3/CUnit-2.1-3.tar.bz2
+TERMUX_PKG_SHA256=f5b29137f845bb08b77ec60584fdb728b4e58f1023e6f249a464efa49a40f214
+TERMUX_PKG_BREAKS="libcunit-dev"
+TERMUX_PKG_REPLACES="libcunit-dev"
+
+termux_step_pre_configure() {
+	./bootstrap
```

Comment:
Looks like the bootstrap script automatically runs configure at the end, and there's no way to skip it. 

The build fails since it then configures without all the extra configure args that the build system sets.

I think you need to run the bootstrap commands you need manually here, or patch the bootstrap script to not run configure at the end.

Reply:
Removed the `bootstrap` script call and I use directly the instructions to prepare files before the call to `./configure`